src/openforms/config/migrations/0058_migrate_to_order_id_template.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.2.11 on 2024-04-12 10:12
+
+from django.db import migrations
+from django.db.migrations.state import StateApps
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+from django.conf import settings
+
+
+def migrate_to_order_id_template(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    """Migrate from the payment order ID prefix to a customizable template.
+
+    If this migration is run on a new instance, the default value of the model field will take effect.
+    Otherwise, the default value is adapted to have the ``{uid}`` placeholder included.
+    """
+    GlobalConfiguration = apps.get_model("config", "GlobalConfiguration")
+    VersionInfo = apps.get_model("upgrades", "VersionInfo")
+    config = GlobalConfiguration.objects.first()
+    if config is None:
+        return
+
+    version_info = VersionInfo.objects.first()
+    assert version_info is not None
```

Comment:
because we have the system check that creates it, right?

Technically you can end up in a situation without record when system checks are completely disabled. Edge case, but not impossible. In that case however, you can also conclude that it's a fresh installation (because the post-migrate hook has never run).

Reply:
Fixed with the suggestion you gave below